components/brave_wallet_ui/page/screens/swap/hooks/useSwap.ts

```diff
@@ -337,7 +378,7 @@ export const useSwap = () => {
                   .format()
               : '',
           fromToken: params.fromToken.contractAddress,
-          toAccountId,
+          toAccountId: fromAccount.accountId,
```

Comment:
A better place to make this assumption is where `toAccountId` is defined, but only if `toAccountId` is not specified as a query param.

Reply:
Done!